build/ansible/pmm2/post-build-actions.yml

```
@@ -67,25 +67,42 @@
         state: stopped
       ignore_errors: True
 
-    - name: Remove pmm-managed database from postgres
+    - name: Supervisord stop           | Stop supervisord service for AMI/OVF
+      when: ansible_virtualization_type != "docker"
+      service: name=supervisord state=stopped enabled=yes
+
+    - name: Supervisord stop           | Stop supervisord service for docker
+      when: ansible_virtualization_type == "docker"
+      shell: supervisorctl shutdown
+
+      # PMM-11336 - The previous steps failed to start PostgreSQL using supervisord,
+      # so a temporary solution was to start it without supervisord and remove the
+      # pmm-managed database/role. However, a complete overhaul of the pipeline is
+      # necessary for a permanent fix.
+    - name: Start PostgreSQL database without supervisord
+      command: /usr/pgsql-14/bin/pg_ctl start -D /srv/postgres14
+      become_user: postgres
+
+    - name: Remove pmm-managed database
       postgresql_db:
+        login_user: postgres
         name: pmm-managed
         state: absent
-      ignore_errors: True
+      register: db_check_result
+
+    - name: Output db_check_result
+      debug:
+        msg: "pmm-managed database does not exist"
```

```suggestion
        msg: "pmm-managed database was removed"
```
Should it read that way?